### PR TITLE
docs: Update recommendation

### PR DESCRIPTION
**Pull request by tkvern**
Drag & drop hierarchical list made as a react component for Ant Design

[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | Modify document community select component document, add antd-nestable    |
| 🇨🇳 Chinese |  修改文档社区精选组件文档，增加 antd-nestable    |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

-----
[View rendered docs/react/recommendation.zh-CN.md](https://github.com/tkvern/ant-design/blob/patch-1/docs/react/recommendation.zh-CN.md)

### CODE REVIEW

File: docs/react/recommendation.zh-CN.md [hunk 1]
```diff
@@ -37,6 +37,7 @@ title: 社区精选组件
 | 关键字高亮 | [react-highlight-words](https://github.com/bvaughn/react-highlight-words) |
 | 文字轮播 | [react-text-loop](https://github.com/braposo/react-text-loop) |
 | 动画 | [react-move](https://github.com/react-tools/react-move) [Ant Motion](https://motion.ant.design/components/tween-one) [react-spring](https://www.react-spring.io) |
+| 可拖拽列表 | [antd-nestable TS版本可用](https://github.com/tkvern/antd-nestable)
```
Comment (yoyo837) [inline docs/react/recommendation.zh-CN.md:40]: 多语文档一起改

Comment (tkvern) [reply docs/react/recommendation.zh-CN.md]: 好的

Comment (tkvern) [reply docs/react/recommendation.zh-CN.md]: @yoyo837 I was updated doc

**Comment by afc163** [comment]
抱歉，暂时不会收录未经市场验证的新包。

**Pull request closed without merging**